frontend/add-client/confirm.component.tsx

```diff
@@ -75,6 +125,10 @@ export default function Confirm(props: StepComponentProps) {
           {props.clientState.lastName} to database.
         </div>
       </div>
+      <ViewEditBasicInfo client={viewableClient} editable={false} />
```

Comment:
Reusing the components from the client record to show the info on the confirm page.